jwst/jump/jump_step.py

```diff
@@ -36,9 +36,11 @@ class JumpStep(Step):
         use_ellipses = boolean(default=False) # deprecated
         sat_required_snowball = boolean(default=True) # Require the center of snowballs to be saturated
         min_sat_radius_extend = float(default=2.5) # The min radius of the sat core to trigger the extension of the core
-        sat_expand = integer(default=2) # Number of pixels to add to the radius of the saturated core of snowballs
-        find_showers = boolean(default=False) # Turn on shower flagging for MIRI
-        edge_size = integer(default=25) # Size of region on the edges of NIR detectors where a sat core is not required
+        sat_expand = integer(default=2) # Number of pixels to add to the radius of the saturated core of snowballs       
+        edge_size = integer(default=25) # Size of region on the edges of NIR detectors where a sat core is not required    
```

Comment:
I find this comment unclear. The explanation in the documentation is better.
`The distance from the edge of the detector where saturated cores are not required for snowball detection`